app/models/redirection.rb

```diff
@@ -15,4 +17,11 @@ def next_url
   def previous_url
     previous.url
   end
+
+  def tag_uri
+    unchanging_hostname = "hotlinewebring.club"
```

Comment:
The hostname is in `ENV["APPLICATION_HOST"]`. How about:

```ruby
hostname = ENV.fetch("APPLICATION_HOST")
```

Then it'll work on staging, too.